addons/web_editor/static/lib/odoo-editor/src/OdooEditor.js

```diff
@@ -2013,6 +2013,7 @@ export class OdooEditor extends EventTarget {
                 this.historyStep(true);
                 this._historyStepsStates.set(peek(this._historySteps).id, 'consumed');
                 setTimeout(() => {
+                    // Line below may steal focus (without timeout)
```

Comment:
@Zinston what I mean by this comment is that the statement below used to steal focus before introducing a timeout in the WysiwygDialog's focus.
I don't know why but it happened for all link created from scratch (/link command without pre-selection) _after_ the first one (for which the statement below always executed earlier).
I essentially put a comment here to have your feedback and also point out it's fixing a timeout issue with another timeout, not sure that's a good fix.